spring-rabbit/src/main/java/org/springframework/amqp/rabbit/AsyncRabbitTemplate.java

```diff
@@ -87,6 +87,7 @@
  *
  * @author Gary Russell
  * @author Artem Bilan
+ * @author Java4ye
```

Comment:
Sorry, that's not the name.
We can accept here only your official name.

Reply:
ok，I fixed it.
Thanks.